.github/pull_request_template.yml

```diff
@@ -0,0 +1,25 @@
+name: Pull Request
+description: Pull request template
+title: "[PR]: "
```

Comment:
I don't think a PR prefix is needed here, they'll always appear in the pull request tab, unless this is helpful for the project view?